src/containers/Tenant/QueryEditor/Issues/Issues.tsx

```diff
@@ -0,0 +1,168 @@
+import * as React from 'react';
+import cn from 'bem-cn-lite';
+
+import {Button, Icon, ArrowToggle} from '@yandex-cloud/uikit';
+import ShortyString from '../../../../components/ShortyString/ShortyString';
+
+import {IssueType, SEVERITY, getSeverity} from './models';
+
+import fatalIcon from '../../../../assets/icons/circle-xmark.svg';
+import errorIcon from '../../../../assets/icons/triangle-exclamation.svg';
+import warningIcon from '../../../../assets/icons/circle-exclamation.svg';
+import infoIcon from '../../../../assets/icons/circle-info.svg';
+
+import './Issues.scss';
+
+const blockWrapper = cn('kv-result-issues');
+const blockIssues = cn('kv-issues');
+const blockIssue = cn('kv-issue');
+
+type DataIssues = {
+    error: IssueType;
+    issues?: IssueType[];
+};
+
+interface ResultIssuesProps {
+    data: DataIssues | string;
+    className: string;
+}
+
+export default function ResultIssues({data, className}: ResultIssuesProps) {
+    const [showIssues, setShowIssues] = React.useState(false);
+
+    const hasIssues = typeof data === 'string' ? false : Array.isArray(data?.issues);
+
+    const renderTitle = () => {
+        let content;
+        if (typeof data === 'string') {
+            content = data;
+        } else {
+            const severity = getSeverity(data?.error?.severity);
+            content = (
+                <React.Fragment>
+                    <IssueSeverity severity={severity} /> <span className={blockWrapper('error-message-text')}>{data?.error?.message}</span>
+                </React.Fragment>
+            );
+        }
+
+        return content;
+    };
+
+    return (
+        <div className={blockWrapper()}>
+            <div className={blockWrapper('error-message')}>
+                {renderTitle()}
+                {hasIssues && (
+                    <Button view="normal" onClick={() => setShowIssues(!showIssues)}>
+                        {showIssues ? 'Hide details' : 'Show details'}
+                    </Button>
+                )}
+            </div>
+            {hasIssues && showIssues && (
+                <Issues issues={(data as DataIssues).issues!} className={className} />
+            )}
+        </div>
+    );
+}
+
+interface IssuesProps {
+    className?: string;
+    issues: IssueType[];
+}
+export function Issues({issues, className}: IssuesProps) {
+    const mostSevereIssue = issues.reduce((result, issue) => {
+        const severity = issue.severity ? issue.severity : 10;
+        return result < severity ? result : severity;
+    }, issues[0].severity ?? 10);
```

Comment:
10 is more than enough here, considering you're scanning through all issues, including issues[0].